.github/workflows/lint_test_build.yml

```diff
@@ -1,14 +1,17 @@
+name: Lint - Test - Build
+
 on:
   push:
     branches:
-      - '!main'
+      - main
```

Comment:
do we want to keep running this on `main` still even if it has overlapping steps with `node-build` action?

Reply:
No, ofc not. Apparently during rebasing I forgot to remove this file. This file is not needed anymore. I will remove it